main_2024.py

```diff
@@ -374,7 +412,16 @@ def main() -> int:
                 main_logger.debug(
                     "geolocation confidence: " + str(detection_world.confidence), True
                 )
-
+        try:
+            cluster_estimations = cluster_estimation_to_main_queue.queue.get_nowait()
+        except queue.Empty:
+            cluster_estimations = None
+        if cluster_estimations is not None:
+            for cluster in cluster_estimations:
+                main_logger.debug("Cluser in world: True")
```

Comment:
Fix spelling mistake with cluster. Also `True` is a parameter, not a part of the string literal.